scene-lib/src/scenelib/annotations/el/AnnotationDef.java

```diff
@@ -73,6 +78,19 @@ public static AnnotationDef fromClass(Class<? extends java.lang.annotation.Annot
         }
 
         Map<String,AnnotationFieldType> fieldTypes = new LinkedHashMap<>();
+        // Preserves order of members, which getDeclaredMethods does not do.
```

Comment:
not sure i follow - this is the only call to MethodRecord.getMethods() in the source.  what is your thinking here?

Reply:
Yes, this is the only call.  I was thinking it would clarify the logic of this method, to get the details of the computation of `methods` out of it.  It's not essential.